src/lib.rs

```diff
@@ -0,0 +1,23 @@
+pub mod rng;
+
+/// Simple example of a function that returns a string
+/// 
+/// # Example
+/// 
+/// ```
+/// use libelinvar::hello;
+/// hello();
+/// ```
+pub fn hello() -> String {
+    return "Elinvar".to_owned();
+}
+
+#[cfg(test)]
```

Comment:
From the point of view of the Rust Modules system, this is a separate module. From the point of the file system, I believe that it's recommended that the Unit Tests are in the same file as what is being tested, whilst Integration Tests are separate - since they will by definition test a larger amount of code.